### PR TITLE
extending get_profile(id) to include professional interests

### PR DESCRIPTION
**Pull request by TS404**
See #94

### CODE REVIEW

File: R/scholar.r [hunk 1]
```diff
@@ -28,7 +28,7 @@ utils::globalVariables(c("name"))
 ##' @export
 ##' @importFrom stringr str_trim str_split
 ##' @importFrom xml2 read_html
-##' @importFrom rvest html_table html_nodes html_text
+##' @importFrom rvest html_table html_nodes html_text html_children
```
Comment (jefferis) [inline R/scholar.r:31]: Thanks @TS404 but you need to regenerate and commit the `NAMESPACE` file also for this import to be effective.

File: R/scholar.r [hunk 2]
```diff
@@ -17,7 +17,7 @@ utils::globalVariables(c("name"))
 ##' multiple scholars.
 ##'
 ##' @return 	a list containing the scholar's name, affiliation,
-##' citations, impact metrics, fields of study, homepage and
+##' citations, impact metrics, research intersts, homepage and
```
Comment (GuangchuangYu) [inline R/scholar.r:20]: pls also fix typo.

**Comment by TS404** [comment]
Thanks for the notes above! I've now submitted the updated pull request #98

**Comment by jefferis** [comment]
Thanks for the contribution @TS404! However submitting a second PR (#98) on top of a first PR (#95) to fix something small that was missing probably isn't the best option. Furthermore you were still missing the corresponding Rd file from your commit in #98. I have added the missing files in 9add58c. Note that this PR is currently failing for unrelated reasons due to a Google change addressed by #97.

**Pull request merged**